svc-events/README.md

```diff
@@ -1,15 +1,27 @@
 # Events
 
-Aggregator for ODIM offers an event interface that allows northbound clients to interact and receive notifications such as alerts and alarms from multiple s, including Aggregator for ODIM itself. It exposes Redfish `EventService` APIs for managing events.
+Resource Aggregator for ODIM offers an event interface that allows northbound clients to interact and receive notifications such as alerts and alarms from multiple resources, including Resource Aggregator for ODIM itself. It exposes Redfish `EventService` APIs for managing events.
 
 An event is a way of asynchronously notifying the client of some significant state change or error condition, usually of a time critical nature.
 
 Use these APIs to subscribe a northbound client to southbound events by creating a subscription entry in the service.
 
+**Supported endpoints**
+
+|API URI|Operation Applicable|Required privileges|
+|-------|--------------------|-------------------|
+|/redfish/v1/EventService|GET|`Login` |
+|/redfish/v1/EventService/Subscriptions|POST, GET|`Login`, `ConfigureManager` |
```

Comment:
Please recheck the required privileges for the endpoints

Reply:
I have rechecked and made changes accordingly.